src/randomize.jl

```diff
@@ -21,14 +21,21 @@ Always shuffles the order of the zero-mass compounds.
 function sort_biosystem_compounds(
     compound_structs::Compounds,
     biosystem_compounds::IDs,
+    tids::IDs,
     sortkey::Symbol=:exact_mass,
     zero_mass_behavior::String="end")
```

Comment:
We might want to just get rid of this keyword argument since it's not doing anything anymore

Reply:
Sure, I deleted it